frontend/src/components/header.js

```diff
@@ -42,65 +54,84 @@ BootstrapNavLink.propTypes = {
  * @param {object[]} props.routes
  * @returns
  */
+
 export function Header(props) {
+    let location = useLocation();
+    let currentRoute = getRoute(location.pathname);
     const { routes = [], infoComponents = null } = props;
 
+    const activeTabs = routes
+        .filter((route) => route.hidden !== true)
+        .map((route) => {
+            const active = currentRoute === route.route.substring(1);
+            return (
+                <BootstrapNavItem
+                    eventKey={route.route}
+                    to={route.route}
+                    key={route.route}
+                >
+                    <h5
+                        className={
+                            active
+                                ? "font-weight-bold text-primary"
```

Comment:
This is the same as embedding styles directly...come up with an appropriate class name, e.g., `nav-link.active` and use that.